ibl_to_nwb/updated_conversion/brainwide_map/convert_brainwide_map.py

```diff
@@ -89,21 +89,26 @@ def convert_session(base_path: Path, session: str, nwbfile_path: str, stub_test:
         conversion_options=conversion_options,
         overwrite=True,
     )
-    automatic_dandi_upload(dandiset_id="000409", nwb_folder_path=nwbfile_path.parent, cleanup=True)
+    if not stub_test:
+        automatic_dandi_upload(dandiset_id="000409", nwb_folder_path=nwbfile_path.parent, cleanup=True)
```

Comment:
I don't really like this being part of the convert_session function. It's not clear from the name of the function that the data will be uploaded, and someone might want to do a conversion without upload. Could you make this a separate function or at least make a flag on the current function with default=False?

Reply:
fixed